app/src/main/java/de/tum/in/tumcampusapp/database/repository/RemoteRepository.kt

```diff
@@ -0,0 +1,10 @@
+package de.tum.`in`.tumcampusapp.database.repository
+
+import de.tum.`in`.tumcampusapp.models.cafeteria.Cafeteria
+import io.reactivex.Observable
+
+interface RemoteRepository {
+
+    fun getAllCafeterias() : Observable<List<Cafeteria>>

```

Comment:
I'd rather not use an interface, when you have a single implementation...

Reply:
good point